app-typescript/components/TreeSelect/TreeSelect.tsx

```diff
@@ -738,7 +733,8 @@ export class TreeSelect<T> extends React.Component<IProps<T>, IState<T>> {
                                     <i className="icon-plus-large"></i>
                                 </button>
                             }
-                            {this.props.dragAndDrop
+
+                            {this.props.sortable
```

Comment:
![image](https://github.com/superdesk/superdesk-ui-framework/assets/2076953/43eee8d7-9e43-4e02-a71d-6f9c42763ee9)
see if you can reduce code duplication even more